react/package.json

```diff
@@ -23,7 +23,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "deploy": "react-scripts build && aws s3 rm s3://samplesheet-check-frontend-onboarding/ --recursive && aws s3 cp ./build s3://samplesheet-check-frontend-onboarding/ --recursive"
+    "deploy": "aws --profile dev s3 rm s3://sscheck-front-end-code-dev/ --recursive && aws --profile dev s3 cp ./build s3://sscheck-front-end-code-dev/ --recursive"
```

Comment:
Instead of `aws --profile dev [...]`, we can rely on `AWS_PROFILE` env var? Otherwise, there is gonna be `deploy-prod: aws --profile prod [...]` target, in tad later! 😉

Reply:
remove at ` commit 654c00e`

Reply:
The current package.json is using 
`    "deploy": "aws s3 rm s3://${REACT_APP_BUCKET_NAME}/ --recursive && aws s3 cp ./build s3://${REACT_APP_BUCKET_NAME}/ --recursive"`
https://github.com/umccr/samplesheet-check-frontend/blob/c3ed8a08a1d2d3b93db0cfac24a253388e3c6fc2/react/package.json#L26

The PR here is focused on the CDK folder instead of the react one. Should have written it down at the PR init :sweat_smile: